components/financial_assistance/app/controllers/financial_assistance/benefits_controller.rb

```diff
@@ -6,6 +6,9 @@ class BenefitsController < FinancialAssistance::ApplicationController
 
     before_action :find_application_and_applicant
     before_action :set_cache_headers, only: [:index]
+    before_action :enable_bs4_layout, only: [:index, :create, :update] if EnrollRegistry.feature_enabled?(:bs4_consumer_flow)
```

Comment:
I think on my PR I had this setup such that create and update set bs4 based on the params. see `conditionally_enable_bs4_layout` in `incomes_controller`

probably fine to keep this here, I just was not 100% if these actions were ever being called from an unsupported page for bs4

Reply:
These actions aren't called from anywhere else, so safe to just do the simple thing here.